nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteProcess.java

```diff
@@ -263,10 +287,14 @@ public void process(final OutputStream flowFileOut) throws IOException {
                         // NB!!! Maybe get(long timeout, TimeUnit unit) should
                         // be used to avoid waiting forever.
```

Comment:
I guess this should be removed now, no? (or al least changed to something explaining the timeout)